src/app/features/rpc-transaction-request/stacks/rpc-stacks-transaction-request.routes.tsx

```diff
@@ -9,16 +9,16 @@ import { NonceEditor } from '@app/features/nonce-editor/nonce-editor';
 import { AccountGate } from '@app/routes/account-gate';
 import { LedgerStacksGate } from '@app/routes/ledger-stacks-gate';
 
-interface GenerateStacksRpcTransactionRequestRoutesArgs {
+interface GenerateRpcStacksTransactionRequestRoutesArgs {
```

Comment:
this new name reads strangely to me

Reply:
Ok, will revert it. I was struggling a bit with naming.